.travis.yml

```diff
@@ -1,18 +1,25 @@
-sudo: required 
-dist: trusty 
+sudo: required
 language: generic
 notifications:
   email:
     on_success: change
     on_failure: always
 env:
   global:
-    - ROS_DISTRO="indigo"
     - UPSTREAM_WORKSPACE=file
-    - ROSINSTALL_FILENAME=.travis.rosinstall
   matrix:
-    - ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - DIST=trusty ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
```

Comment:
remove `DIST`, but add `ROS_DISTRO`

If you really want to specify the Ubuntu version, just set `UBUNTU_OS_CODE_NAME`